### DISCUSSION TITLE
esphome possible

### DISCUSSION THREAD

**Discussion by ghost** [jomjol/AI-on-the-edge-device#657] [Q&A]
Hi

Is it possible to compile the whole project with esphome?

**Comment by jomjol** [comment]
I don't know esphome, therefore I cannot judge. Currently no capacity to check. Anybody else able to support?

**Comment by pixeldoc2000** [comment]
No, this is a custom Firmware. This would need a complete rewrite of the project.

Integration via MQTT is easy, so there is not need.

**Comment by jomjol** [comment]
Then this will not be an option. I will close the discussion therefore.
